src/routes/(settings)/settings/authentication/+page.svelte

```
@@ -1,56 +1,48 @@
 <script lang="ts">
   import { createCurrentUser } from "$lib/client/hooks/useCurrentUser.svelte";
-  import SvelteSeo from "svelte-seo";
   import type { PageProps } from "./$types";
-  import EmailVerificationAlert from "$lib/components/alerts/EmailVerificationAlert.svelte";
   import Button from "$lib/components/ui/button/button.svelte";
+  import SettingsLayout from "$lib/components/settings/SettingsLayout.svelte";
 
   const { data }: PageProps = $props();
 
   const currentUser = createCurrentUser(() => data.auth.currentUserInfo);
```

ESLint

[error] 'currentUser' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)